src/components/Theme/style.css

```diff
@@ -75,6 +75,10 @@
     /* A full screen step will be a video/stream step, with a darker background */
     background-image: url('assets/powered-by-onfido-light.svg');
   }
+
+  .noLogo & {
```

Comment:
Stray `&` here

Reply:
Actually this is equivalent to writing `.noLogo .footer`. I haven't worked with css in a while but the logo wasn't removed if I didn't include the `&` there so I used line 74 as an example and it worked.

Reply:
Ah yeah. My bad. Didn't notice that it's a nested class selector.